src/styles.css

```diff
@@ -27,6 +27,10 @@ body {
   border-right: 1px solid #f0f0f0;
 }
 
+.ant-layout-header {
```

Comment:
still not a fan of this change - is !important needed (generally poor practice?)
is there any way we can keep ant design layout by default?